### ISSUE TITLE
Add Fiji holidays

### ISSUE THREAD

**Issue by arkid15r** [vacanza/holidays#1180]
Links:
  - https://www.fiji.gov.fj/About-Fiji/Public-Holidays
  - https://www.timeanddate.com/holidays/fiji/
  - https://en.wikipedia.org/wiki/List_of_festivals_in_Fiji

**Comment by mcbarin** [comment]
Is there any progress in this?

**Comment by kasya** [comment]
> Is there any progress in this?

I've just finished with [Samoa](https://github.com/vacanza/python-holidays/pull/1914) and I'm going to work on Fiji holidays next.

**Comment by Prateekshit73** [comment]
As suggested by @PPsyrius , I would like to work on this issue.

**Comment by arkid15r** [comment]
@Prateekshit73 I know @kasya had started this task but didn't create a PR. Maybe you should combine the efforts or at least reuse the work already done.

@kasya could you share the code you wrote for Fiji?

**Comment by Prateekshit73** [comment]
Hi @kasya,

I hope you're doing well. I wanted to follow up on my previous request regarding the Fiji holidays task. It’s been a month since I reached out, and I haven’t heard back yet. If you could share any code or progress you’ve made, it would really help me move forward with this.

Thank you!

**Comment by kasya** [comment]
Hi @Prateekshit73 ! Sorry for a long silence from me! Just pushed what I have to a branch in my fork - https://github.com/kasya/python-holidays/tree/add-fiji-holidays
I commented out some tests that were failing in order to push the changes, but once you uncomment those you'll see there are I think 2 tests that are failing, and they are connected to how the holidays are observed for the country. 

Thank you!

**Comment by Prateekshit73** [comment]
Thank you for sharing the branch and your progress on the Fiji holidays! I appreciate your work on this.

I will take a look at the code and the failing tests you mentioned. Once I understand the issues, I’ll work on addressing them. If I have any questions, I’ll be sure to reach out.

Thanks again for your efforts!

**Comment by Prateekshit73** [comment]
As I was reviewing the Fiji holidays, I noticed that some holidays listed on [Wikipedia](https://en.wikipedia.org/wiki/List_of_festivals_in_Fiji) are not present on the [Fiji](https://web.archive.org/web/20190731140325/https://www.fiji.gov.fj/About-Fiji/Public-Holidays) government website. Should we consider including these holidays?

Additionally, since I’m working on your fork, @kasya, should I create the pull request (PR) in your fork or in my own fork of the main repository?

Thanks for your guidance!

**Comment by PPsyrius** [comment]
> As I was reviewing the Fiji holidays, I noticed that some holidays listed on [Wikipedia](https://en.wikipedia.org/wiki/List_of_festivals_in_Fiji) are not present on the [Fiji](https://web.archive.org/web/20190731140325/https://www.fiji.gov.fj/About-Fiji/Public-Holidays) government website. Should we consider including these holidays?

We can use `UNOFFICIAL` for non-official observances if you wish to implement them i.e. BlueSky Fiji Music Festival as mentioned on Wikipedia.

Alternatively, `WORKDAY` works as well if they're at least recognised by the local government in certain capacities i.e. a "Teacher's Day" or "Mother's Day" in other countries that's while officially recognised, isn't a day-off.

**Comment by arkid15r** [comment]
> Additionally, since I’m working on your fork, [@kasya](https://github.com/kasya), should I create the pull request (PR) in your fork or in my own fork of the main repository?

No, create a PR for the main repository. Just keep @kasya as a co-author using commit message trailers.

**Comment by Prateekshit73** [comment]
I noticed that the following holidays are listed as `not a public holiday` on Wikipedia: Maha Shivaratri, Holi, Ram Navami, Hanuman Jayanti, Palm Sunday, Bula Festival, Hibiscus Carnival/Festival, Friendly North Festival, Krishna Janmashtami, Sugar Festival, Coral Coast Festival, Navaratri, and BlueSky Fiji Music Festival.

Should we implement these holidays similarly to how we handle the Maha Shivaratri holiday in Sri Lanka?

Looking forward to your thoughts!

**Comment by KJhellico** [comment]
> I noticed that the following holidays are listed as `not a public holiday` on Wikipedia: Maha Shivaratri, Holi, Ram Navami, Hanuman Jayanti, Palm Sunday, 

Bula Festival, Hibiscus Carnival/Festival, Friendly North Festival, Sugar Festival, Coral Coast Festival - these are local holidays (celebrated in certain cities). I wouldn't add the rest of the non-public holidays either, as it's unlikely to be easy to find any source of their exact dates.
